synapse/http/federation/matrix_federation_agent.py

```diff
@@ -61,6 +61,7 @@ def __init__(
         self,
         reactor,
         tls_client_options_factory,
+        user_agent,
```

Comment:
Could you add `user_agent` to the docstring above?

Reply:
Good call! 👍